t/pt-mysql-summary/pt-2109.t

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env perl
+
+BEGIN {
+   die "The PERCONA_TOOLKIT_BRANCH environment variable is not set.\n"
+      unless $ENV{PERCONA_TOOLKIT_BRANCH} && -d $ENV{PERCONA_TOOLKIT_BRANCH};
+   unshift @INC, "$ENV{PERCONA_TOOLKIT_BRANCH}/lib";
+};
+
+use strict;
+use warnings FATAL => 'all';
+use English qw(-no_match_vars);
+use PerconaTest;
+use Sandbox;
+use DSNParser;
+require VersionParser;
+use Test::More;
+use File::Temp qw( tempdir );
+
+local $ENV{PTDEBUG} = "";
+
+my $dp  = new DSNParser(opts=>$dsn_opts);
+my $sb  = new Sandbox(basedir => '/tmp', DSNParser => $dp);
+my $dbh = $sb->get_dbh_for('master');
+my $has_keyring_plugin;
+
+if ( !$dbh ) {
+   plan skip_all => 'Cannot connect to sandbox master';
+}
+else {
+   plan tests => 3;
+}
+
+my $db_flavor = VersionParser->new($dbh)->flavor();
+if ( $db_flavor =~ m/Percona Server/ ) {
+    my $rows = $dbh->selectall_hashref("SHOW PLUGINS", "name");
+    while (my ($key, $values) = each %$rows) {
+        if ($key =~ m/^keyring_/) {
+            $has_keyring_plugin=1;
+            last;
+        }
+    }
+}
+
+if (!$has_keyring_plugin) {
+    plan skip_all => 'Keyring plugins are not enabled.';
```

Comment:
Do we really need to skip this test if "Keyring" plugin is not enabled?

Reply:
Good point! Removed.